models/business-unit/src/associate/associate-draft/generator.ts

```diff
@@ -0,0 +1,17 @@
+import { ReferenceDraft } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { AssociateRoleAssignmentDraft } from '../../associate-role-assignment/';
+import { TAssociateDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/business-units#associatedraft
+
+const generator = Generator<TAssociateDraft>({
+  fields: {
+    associateRoleAssignments: [
+      fake(() => AssociateRoleAssignmentDraft.random()),
+    ],
+    customer: fake(() => ReferenceDraft.random().typeId('customer')),
```

Comment:
```suggestion
    customer: fake(() => ReferenceDraft.presets.customerReference()),
```

Reply:
fixed in [0c72cd8](https://github.com/commercetools/test-data/pull/481/commits/0c72cd8436aafea8d4e2897fc78208e5bf5ccf05)